src/test/L1CustomERC20Gateway.t.sol

```diff
@@ -178,7 +178,16 @@ contract L1CustomERC20GatewayTest is L1GatewayTestBase {
         address recipient,
         bytes memory dataToCall
     ) public {
-        gateway.updateTokenMapping(address(l1Token), address(l2Token));
+        // message 0 is append here
+        gateway.updateTokenMapping{value: 1 ether}(address(l1Token), address(l2Token));
```

Comment:
why add this step?

Reply:
We only allow the update the mapping in L1 and relay the information L2 now. So the `updateTokenMapping` will result in a cross chain message, which need relay fee.

Reply:
oh, I see. placed the comment in the wrong place.

the original problem is confused about the nonce change from 0 to 1, and it's because `updateTokenMapping` will send a cross-chain message (with nonce 0 in this test).